.env

```diff
@@ -0,0 +1,4 @@
+MONGO_URI=mongodb+srv://edgaras318:Edgaras1@cluster0.i1xgu.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0
+PORT=5001
+JWT_SECRET=d3aed2d849951ecf10896406658d23c06444d14a1be8cab84db325736ba3dde4
```

Comment:
.env fails turi būti paslėptas su .gitignore.
ir sukurtas failas pavadinimu ".env.example" kuriame būtų kintamųjų pavadinimai be reikšmių